rpc/swap.go

```diff
@@ -50,9 +50,11 @@ func NewSwapService(
 }
 
 // PastSwap represents a past swap returned by swap_getPast.
+// TODO: This struct does not have needed fields for swaps of ERC20 tokens.
```

Comment:
such as? maybe add to comment?

Reply:
Sorry, I forgot to remove that TODO. I added `EthAsset` to both `PastSwap` and `OngoingSwap` as part of this PR.